hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/file/FileService.java

```diff
@@ -33,5 +33,5 @@ public interface FileService extends Service {
      */
     @NotNull
     @Override
-    FilePreTransactionHandler createPreTransactionHandler(@NotNull States states);
```

Comment:
Please merge the develop branch in your branch since this interfaces are moved to new modules.

Reply:
Done